webui.go

```diff
@@ -0,0 +1,106 @@
+// See LICENSE.txt for licensing information.
+
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"strconv"
+	"time"
+)
+
+import (
+	"github.com/gorilla/mux"
+)
+
+var webuiLog *log.Logger
+
+func handleGetList(w http.ResponseWriter, r *http.Request) {
+	outputJSON(w, cfg)
+}
+
+func handleGetGameBackup(w http.ResponseWriter, r *http.Request) {
+	g, ok := getGame(w, r)
+	if !ok {
+		return
+	}
+	if err := r.ParseForm(); err != nil {
+		webuiLog.Println("FAILED to parse form")
+		http.Error(w, "Couldn't parse form.", 400)
+		return
+	}
+	note := r.Form.Get("note")
+
+	sv, err := g.Backup()
+	if err != nil {
+		webuiLog.Println("FAILED to save: ", err)
+		http.Error(w, fmt.Sprintln("Failed to save: ", err), 501)
+		return
+	}
+	if note != "" {
```

Comment:
Hmm, maybe not needed? If `note` is not `omitempty` it will be a zero-string anyway.